pawn/Resources/Games/FightClub.pwn

```diff
@@ -1071,8 +1093,9 @@ CFightClub__EndMatch(matchid, deathPlayerId)
 
     if(Player(iPlayer1)->isConnected())
     {
-        SetPlayerVirtualWorld(iPlayer1, 0);
-        SetPlayerInterior(iPlayer1, 0);
+        SetPlayerPos(iPlayer1, FighterPos[iPlayer1][0], FighterPos[iPlayer1][1], FighterPos[iPlayer1][2]);
+        SetPlayerVirtualWorld(iPlayer1, fighterworld[iPlayer1]);
+        SetPlayerInterior(iPlayer1, fighterint[iPlayer1]);
         ResetWorldBounds(iPlayer1);
 
         if (iPlayer1 != deathPlayerId) {
```

Comment:
There's a `SpawnPlayer` call immediately after here, wouldn't that break it?

The spectate change of this PR looks good, but I think that the adjustment for fighters is broken. I think that the existing `SetPlayerVirtualWorld` and `SetPlayerInterior` calls were also broken.